Embedded Metadata.js

```diff
@@ -865,7 +877,7 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"date": "2011/10/17",
+				"date": "2011",
```

Comment:
this is a change in site data; comes straight from citation_data, so unrelated to these changes.